README.md

```
@@ -75,7 +75,8 @@ The most contributed Agent Skills repository, built and maintained together with
 | [Skills by Figma](#skills-by-figma) | [Marketing Skills by Corey Haines](#marketing-skills-by-corey-haines) | [Skills by Binance](#skills-by-binance) |
 | [Product Manager Skills by Dean Peters](#product-manager-skills-by-dean-peters) | [Product Management Skills by Paweł Huryn](#product-management-skills-by-pawel-huryn) | [Skills by MiniMax](#skills-by-minimax-team) |
 | [Skills by DuckDB](#skills-by-duckdb) | [Skills by GSAP](#skills-by-gsap-greensock) | [Skills by Garry Tan (gstack)](#skills-by-garry-tan-gstack) |
-| [Skills by Notion](#skills-by-notion) | [Skills by Resend](#skills-by-resend) | [Community Skills](#community-skills) |
+| [Skills by Monte Carlo Data](#skills-by-monte-carlo-data) | [Skills by Notion](#skills-by-notion) | [Skills by Resend](#skills-by-resend) |
+| [Community Skills](#community-skills) |
```

markdownlint-cli2 (0.22.0)

[warning] Table column count
Expected: 3; Actual: 1; Too few cells, row will be missing data

(MD056, table-column-count)